### PR TITLE
Prefix dtc overlay with image

### PR DESCRIPTION
**Pull request by hakonfam**
Signed-off-by: Håkon Øye Amundsen <haakon.amundsen@nordicsemi.no>

### CODE REVIEW

File: boot/zephyr/CMakeLists.txt [hunk 1]
```diff
@@ -13,16 +13,16 @@ set(BOARD qemu_x86)
 # and fits inside, the boot partition. (If the user specified a
 # DTC_OVERLAY_FILE on the CMake command line, we need to append onto
```
Comment (hakonfam) [inline boot/zephyr/CMakeLists.txt:14]: Should we state `${IMAGE}DTC_OVERLAY_FILE` or even `mcuboot_DTC_OVERLAY_FILE` ?

IMO it is not needed as how to set this variable is documented elsewhere, and here we just need the motivation for the code.

Comment (SebastianBoe) [reply boot/zephyr/CMakeLists.txt:14]: Agreed.

**Comment by SebastianBoe** [comment]
The commit message needs a tag.

EDIT: Ah, this is going upstream, then it should not have one.

**Comment by hakonfam** [comment]
My idea was to get it squashed to the upstream PR commit, then to create separate downstream patch with the changes taken in to the upstream PR

**Pull request merged**